scripts/deleteExpiredAccessTokens.js

```diff
@@ -157,23 +170,40 @@ gpii.accessTokens.flush = function (options) {
 };
 
 /**
- * Create and execute the steps to delete the access tokens.
+ * Create and execute the steps to delete the access tokens recursively.
+ * @param {Object} options - All docs URL and whether to filter:
+ * @param {Array} options.accessTokensUrl - The url for retrieving all of the
+ *                                          access tokens in the database.
+ * @param {Array} options.postOptions - The url for posting the bulk deletion.
+ * @param {Number} options.totalDeleted - The total number of deleted access tokens.
  */
-gpii.accessTokens.deleteAccessTokens = function () {
-    var options = gpii.accessTokens.initOptions(process.argv);
+gpii.accessTokens.deleteRecursive = function (options) {
     var sequence = [
         gpii.accessTokens.retrieveExpiredAccessTokens,
         gpii.accessTokens.flush
     ];
     fluid.promise.sequence(sequence, options).then(
         function (/*result*/) {
-            fluid.log("Done.");
-            process.exit(0);
+            gpii.accessTokens.deleteRecursive(options);
         },
         function (error) {
-            fluid.log(error);
-            process.exit(1);
+            if (error.errorCode === "GPII-NO-MORE-DOCS") {
+                console.log("Done: " + error.message + " Deleted " + options.totalDeleted + " expired access tokens in total.");
+                process.exit(0);
```

Comment:
Similarly - try to have directives to exit the process at the very highest level - better to return a promise from this function